Jenkinsfile

```diff
@@ -20,6 +20,7 @@ pipeline {
         SOCOK8S_DEVELOPER_MODE = "True"
         DEPLOYMENT_MECHANISM = "openstack"
         ANSIBLE_STDOUT_CALLBACK = "yaml"
+        ANSIBLE_DEBUG = "True"
```

Comment:
Do we really need this?  Things will become hardly readable.

I expect that ``ANSIBLE_VERBOSITY=2`` (or 1) would be enough for most cases

Reply:
This was just to catch the problem with CI jenkins jobs failing to reboot

Reply:
sounds good. We should have a look at pipelining too, and enable it by default in gates if it's not yet done.